ansible/roles/ocp-workload-aiadv-fuseonline/readme.adoc

```diff
@@ -0,0 +1,33 @@
+= ocp-workload-fuse-ignite
+
+Login into a target OCP environment as a cluster admin.
+
+Afterwards, execute these commands with these env variables
+
+== Execution using oc client against remote host
+
+GUID=3d5k
+
+OCP_USERNAME=`oc whoami`
+SUBDOMAIN_BASE=`oc whoami --show-server | cut -d'.' -f 2,3,4,5 | cut -d':' -f 1`
+WORKLOAD="ocp-workload-fuse-ignite"
```

Comment:
same here

Reply:
@fridim Thanks, addressed using this commit b05b047a4ec79cdccea9a9ee7c0f2eada88d4222